coursework-template.Rmd

```diff
@@ -179,37 +178,72 @@ You could pick one of these topics:
+    **Data processing: 20%**
 
-1. The selection and effective use of input datasets that are large (e.g. covering multiple years), complex (e.g. containing multiple variables) and diverse (e.g. input datasets from multiple sources are used and where appropriate combined in the analysis)
-1. Describe how the data was collected and implications for data quality, and outline how the input datasets were downloaded (with a reproducible example if possible)
-1. Evidence of data cleaning techniques and adding value to the dataset (e.g. by creating new variables or by joining two datasets)
-1. Use of transport planning techniques such as use of buffers, scenarios, origin-destination data analysis geographic desire lines and routing
+1. The selection and effective use of input datasets that are large (e.g. covering multiple years), complex (e.g. containing multiple variables) and/or diverse (e.g. input datasets from multiple sources are used and where appropriate combined in the analysis)
+1. Describe how the data was collected and implications for data quality, and outline how the input datasets were downloaded (with a reproducible example if possible), with a description that will allow others to understand the structure of the inputs and how to import them
+1. Evidence of data cleaning techniques (e.g. by re-categorising variables)
+1. Adding value to datasets with joins (key-based or spatial), creation of new variables (also known as feature engineering) and reshaping data (e.g. from wide to long format)
 
-    **Choice of topic and impact**
+**Distinction (70%+):** The report makes use of a complex (with many columns and rows) and/or multiple input datasets, efficiently importing them and adding value by creating new variables, recategorising, changing data formats/types, and/or reshaping the data. Selected datasets are appropriate for the research questions, clearly described, with links to the source and understanding of how the datasets were generated.
 
-1. Topic selection, including originality, availability of datasets related to the topic and relevance to solving transport planning problems
-1. Discuss alternative topics considered and justify the topic
-1. Make specific recommendations for further research or policies that are supported by the data analysis undertaken
-1. Explain the potential impacts of the work
+**Merit (60-69%):** Between pass and distinction.
+
+**Pass (50-59%):** The selection, description of, cleaning and value-added to the input datasets are weak overall. The report does not make use of complex or multiple input datasets, or the datasets are not appropriate for the research questions, or the datasets are not clearly described, or there are no links to the source or understanding of how the datasets were generated.
+
+**Fail (0-49%):** The report does not make use of appropriate input datasets and contains very little or now evidence of data cleaning, adding value to the datasets or reshaping the data.
```

Comment:
This is negative description. I.e. just says don't meet pass criteria. Need some positive description of what the student can do and still fail.

Reply:
> While there may be some evidence of data processing, it is of poor quality and/or not appropriate for the research questions.

Added.